themes/digital.gov/layouts/shortcodes/img.html

```diff
@@ -1,3 +1,22 @@
+{{/* =======
+
+  Displays an image with optional metadata
+  See wiki for more details https://github.com/GSA/digitalgov.gov/wiki/Hugo-shortcodes
+  TODO: finish documenting the other properties of this component
```

Comment:
@lcummings12 is this an *actual* TODO?

Reply:
Hi @RileySeaburg and @lcummings12 

It can be removed.. but, any time we update Hugo short codes, our Wiki instructions here https://github.com/GSA/digitalgov.gov/wiki/Hugo-shortcodes **must** be updated as well. The section for images is the first one on the page.

 ~ Same goes for any development updates (gulp, etc.) for local installation instructions; those are in the Readme, here https://github.com/GSA/digitalgov.gov?tab=readme-ov-file#development-guide 

When there's a Trello card to make such updates, there should always be a checklist step to update the GitHub Wiki or the Readme's Development section.

I will merge, but pls add that as a step to the card if it isn't already (and if necessary, you can always convert a checklist item to be its own card).

thx!